src/utils/__tests__/actions.spec.js

```diff
@@ -470,35 +470,28 @@ describe('actions', () => {
         translator,
         senseNavigation,
       })();
-      expect(app.createTemporaryBookmark).toNotHaveBeenCalled;
+      expect(app.createTemporaryBookmark).toHaveBeenCalledTimes(0);
     });
 
-    it('should call refreshDynamicViews', async () => {
+    it("should call refreshDynamicViews", async () => {
       senseNavigation.refreshDynamicViews = jest.fn();
-      const actionObject = actions.find((action) => action.value === 'refreshDynamicViews');
+      const actionObject = actions.find((action) => action.value === "refreshDynamicViews");
       await actionObject.getActionCall({ senseNavigation })();
-      expect(senseNavigation.refreshDynamicViews).toHaveBeenCalled;
-    });
-
-    it('should not call refreshDynamicViews if not implemented in sense-client', async () => {
```

Comment:
This test did not make any sense and shows that .toNotHaveBeenCalled is not a correct assertion. senseNavigation.refreshDynamicViews is not a mock. I don't think we can check this test case because the code checks if refreshDynamicViews is a function before calling it. That won't work when you have to set it to a jest mock (it will automatically be a function)